Source/UI/NewUI/PaymentForm/ViewController/PaymentViewController.swift

```diff
@@ -148,6 +154,8 @@ final class PaymentViewController: UIViewController {
 extension PaymentViewController {
   private func setupViewModel() {
     delegate = self.viewModel as? PaymentViewControllerDelegate
+    view.backgroundColor = viewModel.paymentFormStyle?.backgroundColor
+    stackView.backgroundColor = viewModel.paymentFormStyle?.backgroundColor
```

Comment:
this method is called `setupViewModel`, but we're setting up the view instead. Can we find a more logical location for this code?